docs/sections/contributor_guide/developer_setup.rst

```diff
@@ -1,20 +1,20 @@
 Developer Setup
 ===============
 
-Creating a ``bash`` development shell
+Creating a ``bash`` Development Shell
 -------------------------------------
 
 .. include:: ../../shared/miniforge3_instructions.rst
 
 2. Install the :anaconda-condev:`condev package<>` into the base environment.
 
-  .. code:: sh
+  .. code-block:: text
```

Comment:
I've tried up update any deprecated `code::` directives to `code-block::`, and have tried to set language hints sensibly.

In re: `sh` -> `text`, although many of these code blocks result from running code in a shell, the output is not actually _sh code_ and shouldn't really be formatted as such. For example, the word "set" is a `sh` keyword, so when it appears in some tool output, Sphinx colors it green, even though it's not actually shell code in that context. If we've got actual `sh` / `bash` _code_, we should `code-block:: sh`.